mithril-test-lab/mithril-end-to-end/src/mithril/infrastructure.rs

```diff
@@ -23,12 +23,14 @@ pub struct MithrilInfrastructure {
 }
 
 impl MithrilInfrastructure {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Should this be merged that way?

Reply:
Introduced a new config structure to avoid the clippy hack :+1: